extensions/wikia/JWPlayerTag/JWPlayerTagController.class.php

```diff
@@ -38,13 +38,14 @@ public function renderTag( $input, array $args, Parser $parser, PPFrame $frame )
 			RequestContext::getMain()->getTitle()->getArticleID()
 		) ) {
 			$script = JSSnippets::addToStack( [
-				'jwplayer_tag_ads_js',
-				'/extensions/wikia/JWPlayerTag/scripts/jwplayertag.js'
+				'jwplayer_tag_with_featured_video_js',
+				'jwplayer_tag_scss'
```

Comment:
why are you adding `_scss` file with a function called JSSnippets?

Reply:
no idea, but it works! ;)